lib/internal/url.js

```diff
@@ -219,6 +219,14 @@ class URL {
     if (base !== undefined && !(base instanceof URL))
       base = new URL(String(base));
     parse(this, input, base);
+
+    Object.defineProperty(this, Symbol.toStringTag, {
+      value: 'URL',
+      writable: false,
+      enumerable: false,
+      configurable: true
+    });
```

Comment:
Yeah, I'd considered doing that but just went with the style already implemented for the other uses of `toStringTag` already in the file. We should update all of them if we're going to switch